JSDemos/Demos/TabPanel/SortableClosableTabs/description.md

```diff
@@ -1,18 +1,15 @@
 This demo illustrates how you can allow end-users to reorder, add, and remove tabs within the DevExtreme [TabPanel](/Documentation/ApiReference/UI_Widgets/dxTabPanel/) widget.     
 
-**Reorder Items**
-Wrap the **TabPanel** into the [Sortable](/Documentation/ApiReference/UI_Widgets/dxSortable/) widget and set the following options as needed:
-
+**Reorder Items**       
+- Wrap the **TabPanel** into the [Sortable](/Documentation/ApiReference/UI_Widgets/dxSortable/) widget and set the following options as needed:
```

Comment:
It looks like this item should not be a part of the list:
Wrap the **TabPanel** into the [Sortable](/Documentation/ApiReference/UI_Widgets/dxSortable/) widget and set the following options as needed:
 - step 1
 -step 2
Now it looks different (at least on GitHub)

Reply:
Right you are